src/object/cli_shard.c

```diff
@@ -1288,8 +1284,8 @@ dc_obj_shard_punch(struct dc_obj_shard *shard, enum obj_rpc_opc opc,
 		opi->opi_shard_tgts.ca_arrays = NULL;
 	}
 	uuid_copy(opi->opi_pool_uuid, pool->dp_pool);
-	uuid_copy(opi->opi_co_hdl, args->pa_coh_uuid);
-	uuid_copy(opi->opi_co_uuid, args->pa_cont_uuid);
+	uuid_copy(opi->opi_co_hdl, shard->do_co->dc_cont_hdl);
+	uuid_copy(opi->opi_co_uuid, shard->do_co->dc_uuid);
```

Comment:
just curious that it should be same as original code?

Reply:
Because of space limitation in related data structure, po_{coh,cont}_uuid are not there any longer.